egs/babel_multilang/s5/local/nnet3/run_shared_ivector_extractor.sh

```diff
@@ -12,8 +12,8 @@ stage=4
 suffix=_sp
 feat_suffix=_hires # feat_suffix used in train_set for lda_mllt training.
 nnet3_affix=
-numLeavesMLLT=5500
-numGaussMLLT=90000
+numLeavesMLLT=2500
```

Comment:
@pegahgh, @jtrmal 

I never tested the PCA (vs MLLT) transform in any Babel recipes.

Since we want to switchover to PCA completely, for new recipes I think we might as well make it the default (and potentially only) option.